src/Illuminate/Cache/Console/ClearCommand.php

```diff
@@ -46,13 +47,33 @@ public function __construct(CacheManager $cache)
 	 */
 	public function fire()
 	{
-		$this->laravel['events']->fire('cache:clearing');
+		$storeName = $this->argument('store');
 
-		$this->cache->flush();
+		try {
+			$cacheStore = $this->cache->store($storeName);
+		} catch (\InvalidArgumentException $e) {
```

Comment:
What's the advantage to letting it bubble instead of displaying a succinct message to the user?